automation/test-linux.sh

```diff
@@ -78,8 +82,7 @@ delete_vm(){
   #stop vm
   ./virtctl stop $vm_name -n $namespace
   #delete vm
-  oc process ${template_option} -n $namespace -o json NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
-    oc delete -n $namespace -f -
+  oc delete vm $vm_name -n $namespace
```

Comment:
Can you please replace `oc process` command in windows test too?

Reply:
Done